openwisp_users/tests/test_models.py

```diff
@@ -263,3 +263,25 @@ def test_superuser_has_permission(self):
         user.full_clean()
         user.save()
         self.assertTrue(user.has_permission('not_found.not_found'))
+
+    def test_orguser_is_admin_change(self):
+        org = self._create_org(name='test-org')
+        user1 = self._create_user(username='user1', email='user1@email.com')
+        user2 = self._create_user(username='user2', email='user2@email.com')
+        org_user1 = self._create_org_user(user=user1, organization=org, is_admin=True)
+        org_user2 = self._create_org_user(user=user2, organization=org, is_admin=True)
+
+        with self.subTest('change is_admin when org_user belongs to org_owner'):
+            msg = (
+                f'{user1.username} is the owner of the organization: '
+                f'{org}, and cannot be downgraded'
+            )
+            with self.assertRaisesMessage(ValidationError, msg):
+                org_user1.is_admin = False
+                org_user1.full_clean()
```

Comment:
could you please also use [assertNumQueries](https://docs.djangoproject.com/en/3.1/topics/testing/tools/#django.test.TransactionTestCase.assertNumQueries) to find out how many queries are executed here?